test/FastTests/Server/Replication/ReplicationManualConflictResolution.cs

```diff
@@ -58,8 +77,197 @@ public void CanManuallyResolveConflict()
                         //slave.DatabaseCommands.Put("users/1", null, resolved.DataAsJson, resolved.Metadata);
                     }
                 }
+            }
+        }
+
+
+        [Fact]
+        public void ManuallyResolveToTombstone()
+        {
+            using (var master = GetDocumentStore())
+            using (var slave = GetDocumentStore())
+            {
+
+                SetToManualResolution(master, slave,"return null;");
```

Comment:
Please add in the issue the current state of the script, as in, how it handles all those decisions.